cli/build.rs

```diff
@@ -1,10 +1,9 @@
-use std::fs::{read, write};
+use std::env;
```

Comment:
Is there some convention for import _line grouping_ [*]?
`cargo fmt` sorts, which is great, but I'm not sure if there's prior opinion or work on choosing to group lines together (like here).

Same Q for using `foo.{x, y}` vs `foo.*` vs.
```rust
use foo.x;
use foo.y;
```

Ideally if there's a config file we can write to tell `cargo fmt` what to do about these cases, that'd be great.

Reply:
the main changes i've made here are to group the std imports and to not import `read` and `write` into the namespace, preferring to have to reference them with the `fs` prefix because it improves readability because `read` and `write` are common function names

it may be possible to configure but whenever possible i like to keep the cargo fmt defaults and not grow them further, opting to only enforce so much as it impacts readability